ansible_collections/arista/avd/roles/eos_designs/templates/overlay/ebgp/peer-groups.j2

```diff
@@ -26,3 +26,33 @@
       maximum_routes: 0
       struct_cfg: {{ switch.bgp_peer_groups.evpn_overlay_core.structured_config | arista.avd.default }}
 {% endif %}
+{% if overlay_data.mpls_overlay_peer_group is arista.avd.defined %}
```

Comment:
This peer group is not supported in ebgp scenarios right?

Reply:
Correct, remnants from before dedicated peer group. Removed all references to this peer group from ebgp templates.